Examples/MAX32662/Demo/README.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+This example demonstrates the use of the TFT Display, UART Terminal, RTC, Pushbuttons, and LEDs on the MAX32662EVKIT.
+
+Features:
+	1. Displays the Analog Devices Logo when the example starts.
+	2. Displays the uptime of the board in the format: (hhh:mm:ss).
+	3. LED Toggles at 1Hz by default, or 2Hz if the pushbutton is held down.
+	4. Displays the chip info for 3 seconds if the pushbutton is pressed 5 times.
+
+## Software
+
+### Project Usage
+
+Universal instructions on building, flashing, and debugging this project can be found in the **[MSDK User Guide](https://analog-devices-msdk.github.io/msdk/USERGUIDE/)**.
+
+### Project-Specific Build Notes
+
+Only builds on EV Kits.
```

Comment:
Change to "this project is only supported on the [MAX32662EVKIT](https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/MAX32662EVKIT.html)"